libs/output-mapping/tests/Writer/Helper/SliceCommandBuilderTest.php

```diff
@@ -34,7 +34,7 @@ public function testCreateProcess(): void
             implode(
                 ' ',
                 [
-                    "'./bin/slicer'",
+                    sprintf("'%s/slicer'", $GLOBALS['_composer_bin_dir']),
```

Comment:
Jeste bych tady mozna checknul, ze to `$GLOBALS['_composer_bin_dir']` neni prazdny?